docs/api/browser-window.md

```diff
@@ -1828,14 +1828,19 @@ there is only one tab in the current window.
 
 Adds a window as a tab on this window, after the tab for the window instance.
 
-#### `win.setVibrancy(type)` _macOS_
+#### `win.setVibrancy(type[, options])` _macOS_
 
 * `type` string | null - Can be `appearance-based`, `light`, `dark`, `titlebar`,
   `selection`, `menu`, `popover`, `sidebar`, `medium-light`, `ultra-dark`, `header`, `sheet`, `window`, `hud`, `fullscreen-ui`, `tooltip`, `content`, `under-window`, or `under-page`. See
   the [macOS documentation][vibrancy-docs] for more details.
+* `options` boolean (optional)
+  * `animate` boolean (optional) - Whether to animate or not the vibrancy toggle.
+  * `animationDuration` number (optional) - if nonzero, the change to vibrancy will be animated over the given duration (in milliseconds).
```

Comment:
Sorry, I was unclear—I meant to remove the `animate` parameter altogether and take the cue only from whether `animationDuration` is specified and non-zero.

Reply:
makes sense, removed it!